RustFxA/FxAWebViewController.swift

```diff
@@ -135,7 +135,7 @@ extension FxAWebViewController: WKUIDelegate {
         helpBrowser = wv
         helpBrowser?.navigationDelegate = self
 
-        navigationItem.leftBarButtonItem = UIBarButtonItem(title: .BackTitle, style: .plain, target: self, action: #selector(closeHelpBrowser))
+        navigationItem.leftBarButtonItem = UIBarButtonItem(title: .KeyboardShortcuts.Back, style: .plain, target: self, action: #selector(closeHelpBrowser)) // Also used a keyboard shortcut string that wasn't meant to be used in this manner. Need to fix this by creating a brand new string.
```

Comment:
What do we do about this? Continue using an out of context string, or create a new string?